src/components/Avatar/Avatar.test-int.js

```diff
@@ -0,0 +1,104 @@
+/* global Feature, Scenario */
+const searchWithinIframe = require('../../../test/codecept/utils/searchWithinIframe.js');
+
+const KIND = 'Avatar.Avatar';
+
+Feature('Avatar');
+
+Scenario('Overview', (I) => {
+    searchWithinIframe(I, KIND, 'Overview', () => {
+        I.see('Avatar: Avatar');
+        I.see('Overview');
+        I.see('Example');
+        I.seeElement('.uir-avatar');
+        I.see('DS');
+        I.see('Source Code');
+        I.seeElement('code');
+        I.see('Props');
```

Comment:
I wonder if we're running too many assertions in our integration tests?

Some of the steps looks like they're only testing Storybook (like `I.see('Props');`, and I'm not sure if we need to test each property, if we've already got a unit test for it.

The main concern is that the Avatar integration suite takes 50s to run - so 10 similar sized components would take a simple component library 5-10 minutes to run a full suite